pages/cards/overview/minor-arcana/index.js

```diff
@@ -0,0 +1,49 @@
+import Link from "next/link";
+import { useState, useEffect } from "react";
+import BackButton from "../../../../components/Backbutton/backbutton";
+
+export default function Description() {
+  const [hasMounted, setHasMounted] = useState(false);
+
+  useEffect(() => {
+    setHasMounted(true);
+  }, []);
+  if (!hasMounted) {
+    return null;
+  }
+  return (
+    <>
+      <ul>
+        <h1>minor arcana</h1>
+        <li>
+          {" "}
+          <Link href={{ pathname: "/cards/overview/minor-arcana/wands" }}>
```

Comment:
I think the pathnames should be generated dynamically as well as the corresponding pages to which they link.

Reply:
@PReichetanz done